.github/actions/publish-libs/action.yml

```
@@ -0,0 +1,115 @@
+name: "Publish ACA libs to NPM and GH registry"
+description: "Publish ACA libs to NPM and GH registry"
+
+inputs:
+  github_token:
+    description: 'Github token'
+    required: true
+    type: string
+  npm-registry-address:
+    description: 'NPM registry address'
+    required: true
+    type: string
+  npm-registry-token:
+    description: 'NPM registry token'
+    required: true
+    type: string
+  dry-run:
+    description: dry run flag
+    required: true
+    type: boolean
+
+runs:
+  using: "composite"
+  steps:
+    - name: update libs version
+      shell: bash
+      run: |
+        DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+        ROOT_DIR=${DIR}/../../..
+        DIST_DIR=${ROOT_DIR}/dist/@alfresco
+        PROJECTS_DIR=${ROOT_DIR}/projects
+        VERSION_IN_PACKAGE_JSON=`node -p "require('$ROOT_DIR/package.json')".version;`;
```

```suggestion
        VERSION_IN_PACKAGE_JSON=$(node -p "require('$ROOT_DIR/package.json').version;")
```

the backtick notation is deprecated, always use `$()` when you want to capture the stdout of a command

semicolon is not needed, is equivalent to a new line (it's mandatory when you want multiple commands on a single line)

---

Fixed